mail_tracking_mailgun/models/res_partner.py

```diff
@@ -26,10 +26,13 @@ def _email_bounced_set(self, reason, event):
         for partner in self:
             if not partner.email:
                 continue
-            body = _('Email has been bounced: %s\n'
-                     'Reason: %s\n'
-                     'Event: %s') % (partner.email, reason,
-                                     event and event.id or _('unknown'))
+            body = _(
+                'Email has been bounced: %s\n'
+                'Reason: %s\n'
+                'Event: '
+                '<a href=# data-oe-model=mail.tracking.event data-oe-id=%d>'
+                '%s</a>') % (partner.email, reason, event and event.id or 0,
+                             event and event.id or _('unknown'))
```

Comment:
```suggestion
                             event.id or _('unknown'))
```

Reply:
This is why the https://travis-ci.org/OCA/social/jobs/582600826#L1289 :)

Fixed in the last commit anyway